addons/phone_validation/tools/phone_validation.py

```diff
@@ -22,7 +22,7 @@ def phone_parse(number, country_code):
             ) from e
 
         if not phonenumbers.is_possible_number(phone_nbr):
-            raise UserError(_('Impossible number %s: probably invalid number of digits.', number))
+            raise UserError(_("The number %s has an invalid number of digits, unless you're dialing aliens. Let's fix it for this dimension!", number))
```

Comment:
I don't see the relationship between aliens and dimensions (didn't think I would ever have to comment something like this in a review)

Reply:
Please make an understandable error, moreover while changing this remove the 'invalid number of digits' part, like
```suggestion
            raise UserError(_("The number %s does not appear to be a valid number. [insert bad joke here]", number))
```